CrowServe/CrowServe.cpp

```diff
@@ -1,6 +1,25 @@
 #include "CrowServe.h"
 
 namespace CrowServe {
+    
+bool initialize_sockets()
+{
+    #if Platform == PlatformWindows
```

Comment:
Preprocessor code  should not be intended to the same level as code, typically it's not idented at all, but can be deindented once, depending on style. Also brace style.